tools/validators/instance_validator/validate/field_translation.py

```diff
@@ -18,13 +18,15 @@ class FieldTranslation(object):
   """A translation for a single field of an entity.
 
   Args:
-    field_name: name of the field the translation corresponds to
+    std_field_name: standard name of the field in the ontology
+    raw_field_name: name of the field sent in the telemetry payload
     units: dictionary from standard units to expected telemetry units
     states: dictionary from standard states to expected telemetry states
   """
 
-  def __init__(self, field_name, units, states):
+  def __init__(self, std_field_name, raw_field_name, units, states):
```

Comment:
can we use type hints or explain the data type in the header comment?  Specifically, I was thinking about field names, which are often tuples in the ontology validator, and it wasn't immediately obvious if a imilar patter was being used here.  (Note: it might make sense to share the same tuple type)

Reply:
yes, I added a todo in this file, the next PR I am refactoring the code to extract the yaml parsing, and validation to delegate/include in each class. Will do a pass on the types.

Reply:
bugid? (or issue ID if preferred)

also arg description should begin with the data type.  Ex: "String. Standard field name in the ontology"

Reply:
TODO + b/176095085

Reply:
updated the bug with the latest comment and added a todo